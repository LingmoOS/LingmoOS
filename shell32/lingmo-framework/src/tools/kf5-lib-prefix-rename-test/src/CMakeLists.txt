find_package(KF5 REQUIRED KIdleTime ItemModels KWidgetsAddons KWindowSystem KCodecs KArchive KCoreAddons Solid ThreadWeaver KF5GlobalAccel
                          KConfig KAuth KJS KWallet KDBusAddons KI18n KGuiAddons KConfigWidgets
                          KService ItemViews KNotifications KIconThemes KCompletion KJobWidgets Sonnet KTextWidgets XmlGui KCrash
                          KBookmarks KDeclarative KUnitConversion KParts Kross KIO KInit KDocTools KDESu)

find_package(KCoreAddons REQUIRED)

find_package(Solid REQUIRED)

find_package(KDESu REQUIRED NO_MODULE)

find_package(KDeclarative REQUIRED NO_MODULE)

find_package(KDE4Support REQUIRED NO_MODULE)

include(PlasmaMacros)
include(PlasmaMacros.cmake)

target_link_libraries(Foo
                     Qt5::Test
                     KF5::Plasma
                     KF5::KArchive KF5::KCoreAddons
                     KF5::KConfigGui
                     KF5::KI18n
                     KF5::KIOCore
                     KF5::KService
                     KF5::PlasmaQuick
                    )

target_link_libraries(plasma-example-kpart-shell KF5::Plasma KF5::KParts)
