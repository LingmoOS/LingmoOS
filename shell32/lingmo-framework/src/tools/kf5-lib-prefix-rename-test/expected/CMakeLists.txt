find_package(KF5 REQUIRED IdleTime ItemModels WidgetsAddons WindowSystem Codecs Archive CoreAddons Solid ThreadWeaver GlobalAccel
                          Config Auth JS Wallet DBusAddons I18n GuiAddons ConfigWidgets
                          Service ItemViews Notifications IconThemes Completion JobWidgets Sonnet TextWidgets XmlGui Crash
                          Bookmarks Declarative UnitConversion Parts Kross KIO Init DocTools Su)

find_package(KF5CoreAddons REQUIRED)

find_package(KF5Solid REQUIRED)

find_package(KF5Su REQUIRED NO_MODULE)

find_package(KF5Declarative REQUIRED NO_MODULE)

find_package(KF5KDE4Support REQUIRED NO_MODULE)

include(KF5PlasmaMacros)
include(KF5PlasmaMacros.cmake)

target_link_libraries(Foo
                     Qt5::Test
                     KF5::Plasma
                     KF5::Archive KF5::CoreAddons
                     KF5::ConfigGui
                     KF5::I18n
                     KF5::KIOCore
                     KF5::Service
                     KF5::PlasmaQuick
                    )

target_link_libraries(plasma-example-kpart-shell KF5::Plasma KF5::Parts)
